src/azure.js

```diff
@@ -3,7 +3,7 @@ var api         = require('./v1');
 
 api.declare({
   method:     'get',
-  route:      '/azure/:account/table/:table/read-write',
+  route:      '/azure/:account/table/:table',
```

Comment:
Isn't this used anywhere that azure-entities is used without passing in azure creds directly? That seems like a pretty common pattern and one that will be a bit hard to root out all of the occurences of? Otherwise I agree.